app/controllers/inbound_webhooks/beckn_controller.rb

```diff
@@ -21,10 +21,47 @@ def create
   private
 
   def verify_signature
-    # TODO: Add authentication for the Beckn webhook; Waiting on https://github.com/beckn/protocol-server/issues/152
+    return if secret.blank?
+
+    auth_header = request.headers["authorization"]
```

Comment:
Used `authorization` instead of regular `AUthorization` as the PS has `authorization`

https://github.com/beckn/protocol-server/blob/d85c648c1c803f3db7e9d4f0a135bb4a0a62cbc4/src/utils/auth.utils.ts#L225

Reply:
Actually, I learnt that headers are `case-insensitive`